modules/machine-config-overview.adoc

```diff
@@ -43,7 +43,12 @@ The kinds of components that MCO can change include:
 * **config**: Create Ignition config objects (see the link:https://coreos.github.io/ignition/configuration-v3_2/[Ignition configuration specification]) to do things like modify files, systemd services, and other features on {product-title} machines, including:
 - **Configuration files**: Create or overwrite files in the `/var` or `/etc` directory.
 - **systemd units**: Create and set the status of a systemd service or add to an existing systemd service by dropping in additional settings.
-- **users and groups**: Change ssh keys in the passwd section post-installation.
+- **users and groups**: Change SSH keys in the passwd section post-installation.
+
+[IMPORTANT]
+====
+Only changing the SSH key for the `core` user via Machine Configs is supported.
```

Comment:
```suggestion
Only changing the SSH key for the `core` user through machine configs is supported.
```

Reply:
Phrasing seems unclear to me.  "Machine configs support changing SSH keys only for the `core` user"?